cmd/habitatctl/commands/helpers.go

```diff
@@ -39,6 +40,16 @@ func postRequest(reqType string, req, res interface{}) {
 	}
 }
 
+func postFileRequest(reqType string, req, res interface{}, file *os.File) {
+	address := compass.CustomHabitatAPIAddr("localhost", viper.GetString("port")) + ctl.GetRoute(reqType)
```

Comment:
can we instead pass in addresses when we post requests? i feel like this makes the functions more usable for tests as well

Reply:
if we want to be able to hit endpoints in tests, why not use the `habitat_client` functions that we call internally in these post functions?

Reply:
hmm yeah. i feel like there's some redundency with the different types of post functions, but i think its fine for now